server/services/interventionsService.ts

```diff
@@ -0,0 +1,29 @@
+import RestClient from '../data/restClient'
+import logger from '../../log'
+import { ApiConfig } from '../config'
+
+interface Referral {
+  id: string
```

Comment:
i know there is a comment above about this just being example code, but it's a good opportunity to think about what sort of structure these objects will have. will these IDs be uuids, or an incrementing index, or something else? we could make a type alias 

```
type Uuid = string
```

or call the `id` field `uuid`

```interface Referral { uuid: string } ```

to clarify?

Reply:
We were thinking this might be more of a back-end concern, as we'll probably just use this id to make POST requests, so we were thinking for our purposes a `string` type would probably be enough, but maybe we can chat about this once we know a bit more about how our `Referral`s are going to look and be used?

Reply:
Yup, ditto what George said - the exact structure of an ID seems like a backend implementation detail to me.

Reply:
👍🏼